src/main/java/com/bobocode/bibernate/session/SessionImpl.java

```diff
@@ -67,7 +77,7 @@ public <T> Optional<T> find(Class<T> type, Object primaryKey) {
             throw new BibernateException("More than 1 result were found!");
         }
         Optional<T> entity = Optional.of(foundEntities.get(0));
-        persistenceContext.putEntity(primaryKey, entity.get());
+        persistenceContext.putEntity(entity.get(), primaryKey);
```

Comment:
Maybe lets handle somehow a case where optional is empty to avoid NPE? just calling .get() is not a good practice.
for example some .orElseThrow ->BibernateException(....)